File: client-app/src/app/models/patientprofile.ts
import { patientUser, User } from "./user";

export interface patientprofile {
    id:string;
    displayName: string;
    userName: string;
    name: string;
    lastName: string;
    birthDate: string;
    address:string;
    language:string;
    profession:string;
    bio:string;
    image:string;
    
}

export class patientprofile implements patientprofile {
    constructor(user: patientUser) {
        this.id = user.id;
        this.userName = user.userName;
        this.displayName = user.displayName;
        this.image = user.image;
        this.name = user.name;
        this.lastName = user.lastName;
        this.birthDate = user.birthDate;
        this.address = user.address;
        this.language = user.language;
        this.profession = user.profession;
        this.bio = user.bio;
        this.image = user.image;
    }
}

File: Application/Profiles/PatientProfile.cs
using System;

namespace Application.Profiles
{
    public class PatientProfile
    {
        public string Id {get; set;}
       public string DisplayName {get;set;}

        public string UserName {get;set;}
        public string Name { get; set; }

        public string LastName {get; set;}

        public DateTime? BirthDate {get; set;}

        public string Address{get; set;}

        public string Language {get; set;}

        public string Profession { get; set; }
        
        public string Bio { get; set; }

        public string Image {get; set;}
    }
}